portfolio/src/main/webapp/index.html

```diff
@@ -1,19 +1,118 @@
+    <script src="script.js"></script>
+  </head>
+  
+  <body style="background-color:rgb(40, 114, 143)">
+    
+    <!-- Header for the Website -->
+    <div id="header" class="bg4">
+        <div class="flex-container" style="float:left">
+            <div class="button"> <a href="#Home">Home</a></div>
+            <div class="button"> <a href="#About">About</a></div>
+            <div class="button"> <a href="#Gallery">Gallery</a></div>
+        </div>
+    </div>  
+    
+    <!-- Intro/About Page-->
+    <div class="container" id="Home" style="padding: 0px;">                                     <!-- BKG Image Container-->
+        
+        <img src="images/bay_rocks.jpg" id="bg-img">
+        
+        <div class="container-child flex-container">   <!--Flex Container for side by side elements-->
+            <div class="container center" style="flex:3">       <!--Left Element for person desc.-->
+                <img src="images/flourish.png" id="flourish">
+                <div class="text-block" id="greeting">
+                    <p> <tab1>Hallo! My name is</tab1></p>
+                    <h1 style="font-size: 96px">Tristram</h1>
+                    <h1 style="font-size: 96px">
+                        <span style="color:rgb(255, 192, 74)"> Dacayan </span>
+                    </h1>                    
+                </div>
+            </div>
+            
+            <div class="container center" style="flex:2">       <!--Right Element for person pic.-->
+                <div class="container-child bg3" style="opacity:0.4"></div>
+                <img src="images/graduation_photo.png" id="profile-pic">
+
+                <div class="text-block">
+                    <h2 style="text-align: center"> 
+                        <br>Read more 
+                        <a href="#About">
+                            <span style="color:orange">
+                                about me →
+                            </span>
+                        </a>
+                    </h3>
+                </div>
+
+            </div>
+        </div>
+    </div>
+    
+    <!-- About Me Page -->
+    
+    <div class="container bg3" id="About" style="padding: 0px;"> 
+        <div class="flex-container content center">
+            <div class="container" style="flex:2">
+                <img src="images/courthouse.JPG" class="gallery-photo">
+            </div>
+            <div class="container" style="flex:3">
+                <div class="text-block">
+                    <h1 class="right"> About Me </h1>
+                </div>
+                <div class="text-block center" style="width:75%">
+                    <p style="color:rgb(40, 114, 143)">
+                        <tab2> I am a Computer Science Major studying at Kean University. 
+                        Although I have a passion for Software Development, I have 
+                        no projects or achievements to display. However, I am also 

```

Comment:
Thanks for the advice! I'll be sure to change the paragraph since it conflicts with the message I am trying to send (i.e. I plan to add my projects in the future). I'll also look into the formatter since I had a tough time always trying to keep the textboxes center.